client/packages/common/src/ui/layout/tables/components/Header/ColumnPicker.tsx

```diff
@@ -101,13 +73,15 @@ export const ColumnPicker = <T extends RecordWithId>({
           <Typography style={{ fontWeight: 700 }}>
             {t('table.show-columns')}
           </Typography>
-          {Object.values(columns)
+          {columns
```

Comment:
I don't know why this was here -- `columns` is already an array so `Object.values` does nothing as far as I can tell 🤷‍♂️

Reply:
odd - can't see why it would be required.